runtime/proto/corfu_options.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package org.corfudb.runtime;
+option java_package = "org.corfudb.runtime";
+
+import "google/protobuf/descriptor.proto";
+
+// Option tags to be used by the CorfuStore consumers to tag special fields to be detected by Corfu.
+message SchemaOptions {
+    // Secondary keys to be indexed.
+    bool secondary_key = 1;
+    // Version number in metadata field.
+    bool version = 2;
+}
+
+extend google.protobuf.FieldOptions {
+    SchemaOptions schema = 1036;
```

Comment:
Can someone explain to me what the magic number 1036 means here?

Reply:
It is google protobuf's FieldOptions field number that does not collide with an existing number.